src/main/java/org/gridsuite/filter/server/FilterController.java

```diff
@@ -113,25 +113,6 @@ public ResponseEntity<List<AbstractFilter>> getFiltersMetadata(@RequestParam("id
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(service.getFilters(ids));
     }
 
-    @PutMapping(value = "/filters/{id}/replace-with-script")
-    @Operation(summary = "Replace a filter with a script filter")
-    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The filter have been replaced successfully")})
-    public ResponseEntity<AbstractFilter> replaceFilterWithScript(@PathVariable("id") UUID id, @RequestHeader("userId") String userId) {
-        return ResponseEntity.ok()
-            .contentType(MediaType.APPLICATION_JSON)
-            .body(service.replaceFilterWithScript(id, userId));
-    }
-
-    @PostMapping(value = "/filters/{id}/new-script")
-    @Operation(summary = "Create a new script filter from a filter")
-    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The script filter have been created successfully")})
-    public ResponseEntity<AbstractFilter> newScriptFromFilter(@PathVariable("id") UUID filterId,
-                                                              @RequestParam(required = false, value = "newId") UUID newId) {
-        return ResponseEntity.ok()
-            .contentType(MediaType.APPLICATION_JSON)
-            .body(service.newScriptFromFilter(filterId, newId));
-    }
```

Comment:
Scripting part is obsolete or not used?

Reply:
These endpoints have been removed because, a long time ago, we have decided to remove the possibility to generate a script filter from other kinds of filters ...